Functions/xASL_init_LoadMetadata.m

```diff
@@ -170,10 +170,20 @@
 %xASL_bids_LoadParticipantTSV 
 
     %% Load participants.tsv
-    PathTSV = fullfile(x.D.ROOT, 'Participants.tsv');
+    PathTSV = fullfile(x.D.ROOT, 'participants.tsv');
+    PathTSVold = fullfile(x.D.ROOT, 'Participants.tsv');
+    bParticipantsTSV = false;
+
     if exist(PathTSV, 'file')
-        fprintf('Participants.tsv (BIDS) detected, loading...\n');
         CellArray = xASL_tsvRead(PathTSV);
+        bParticipantsTSV = true;
+    elseif exist(PathTSVold, 'file')
+        CellArray = xASL_tsvRead(PathTSV);
+        bParticipantsTSV = true;
+    end
+
+    if bParticipantsTSV

```

Comment:
especially since the 'participatns.tsv' is reported here.

Reply:
2b189c8 👍